rest/serializers.py

```diff
@@ -97,3 +103,49 @@ class Meta:
             'id',
             'slug',
         )
+
+
+class GroupSerializer(Serializer):
+    """
+    Serializer for base_group_type
+    """
+    # pylint: disable=abstract-method
```

Comment:
No strong opinions about this, but I guess we want a larger consensus in this case.